setup.py

```diff
@@ -1,57 +1,46 @@
 #! /usr/bin/env python
 
+import re
 from setuptools import setup, find_packages
+from typing import List
 
-# Please make sure to cap all dependency versions, in order to avoid unwanted
-# functional and integration breaks caused by external code updates.
 
-general_requirements = [
-    'dpath >= 1.5.0, < 2.0.0',
-    'pytest >= 4.4.1, < 6.0.0',  # For openfisca test
-    'numpy >= 1.11, < 1.21',
-    'psutil >= 5.4.7, < 6.0.0',
-    'PyYAML >= 3.10',
-    'sortedcontainers == 2.2.2',
-    'numexpr >= 2.7.0, <= 3.0',
-    ]
+def require(filename: str) -> List[str]:
```

Comment:
Using `require` as a local function name in a requirements file makes for unclear declarations below.
```suggestion
def load_requirements_from_file(filename: str) -> List[str]:
```